airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingDestination.java

```diff
@@ -87,8 +90,14 @@ public JsonNode toJdbcConfig(final JsonNode config) {
   public AirbyteMessageConsumer getConsumer(final JsonNode config,
                                             final ConfiguredAirbyteCatalog catalog,
                                             final Consumer<AirbyteMessage> outputRecordCollector) {
-    return new StagingConsumerFactory().create(outputRecordCollector, getDatabase(config),
-        new SnowflakeInternalStagingSqlOperations(getNamingResolver()), getNamingResolver(), config, catalog);
+    return new StagingConsumerFactory().create(
+        outputRecordCollector,
+        getDatabase(config),
+        new SnowflakeInternalStagingSqlOperations(getNamingResolver()),
+        getNamingResolver(),
```

Comment:
Nitpick. This is not related to this PR. But the `getNamingResolver` method seems useless. We can instead make `namingResolver` in `AbstractJdbcDestination` a projected variable, and reference `namingResolver` directly.